ci/start-console.sh

```diff
@@ -29,14 +29,7 @@ CONSOLE_PORT=${CONSOLE_PORT:=9000}
 INVENTORY_SERVER_HOST=${INVENTORY_SERVER_HOST:="http://localhost:30088"}
 MUST_GATHER_API_SERVER_HOST=${MUST_GATHER_API_SERVER_HOST:="http://localhost:30089"}
 
-echo "Starting local OpenShift console..."
-echo "==================================="
-
-echo "API Server: $BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT"
-echo "Console Image: $CONSOLE_IMAGE"
-echo "Console URL: http://localhost:${CONSOLE_PORT}"
-echo "Inventory server URL: ${INVENTORY_SERVER_HOST}"
-echo "Must gather API server URL: ${MUST_GATHER_API_SERVER_HOST}"
+[[ ${CONSOLE_IMAGE} =~ ^localhost/ ]] && PULL_POLICY="never" || PULL_POLICY=${PULL_POLICY:="always"}
```

Comment:
use `if ... else` notations, it's easier to read

Reply:
I actually prefer the `&& ||` expression for setting a single env var based on a condition so the assignments are compact.

Reply:
readablity is more important than compacness

Reply:
6 of one, half dozen of another

I swapped to the full on if then else structure.